file.go

```diff
@@ -564,6 +564,9 @@ type ValidateOpts struct {
 
 	// AllowZeroBatches allows the file to have zero batches
 	AllowZeroBatches bool `json:"allowZeroBatches"`
+
+	// BypassCompanyIdentificationMatch can be set to skip validation for //TODO
+	BypassCompanyIdentificationMatch bool `json:"bypassCompanyIdentificationMatch"`
```

Comment:
Originally I tried 
```go
	// ...
	BatchControl struct {
		BypassCompanyIdentificationValidation bool
	}
```
But an embedded struct makes the API awkward to use since you have to set it using 
```go
	batcher.SetValidation(&ValidateOpts{
		BatchControl: struct{
			BypassCompanyIdentificationMatch bool,
		}{
		BypassCompanyIdentificationMatch: true,
		}
	})
```

Reply:
That's a good point. I'm okay calling it *Match.